README.md

```diff
@@ -7,209 +7,119 @@
 
 ## Overview
 
-Magic Modules (MM) is a project to auto-generate management libraries (e.g.
-plugins) for third party tools to operate Google Cloud Platform (GCP) products
-natively.
+Magic Modules is a tool we use to autogenerate infrastructure-as-code tools for
+Google Cloud Platform. GCP ["resources"](https://cloud.google.com/docs/overview/#gcp_resources)
+are encoded in a shared data file, and that data is used to fill in
+"Mad Libs"-style templates across each of the tools Magic Modules generates.
 
+They include:
 
-## Motivation
+* Terraform
+* Ansible
+* InSpec
 
-Customers -- large and small -- use many different deployment tools to provision
-and manage their operating environments. Companies like Puppet, Chef, Ansible,
-Hashicorp produce open source tools to meet these needs and typically provide
-enterprise versions and support.
+Not only is Magic Modules a force multiplier for our developers, Magic Modules
+allows us to preemptively solve issues by encoding field-tested learnings about
+the GCP API into each tool; issues solved for one tool will be fixed in each
+other tool.
 
-Once customers invest in these tools they prefer to use a single management
-system (and style). For this reason Google wants to provide integration for
-these tools, so that our customers have an easier path to GCP versus abandoning
-a tool they're already comfortable with. For example Puppet can provision and
-manage VMs in Compute Engine, databases in Cloud SQL, Google Kubernetes Engine
-clusters, and so on.
+## Getting Started with Magic Modules
 
-But given the breadth of products available in Google Cloud Platform, creating
-modules by hand quickly becomes inefficient and costly. It would involve too
-much redundant engineering work and create too much code for a team to maintain.
-There is also the issue of explosion of modules, if you consider for
-comprehensive coverage: <code>
-total\_libraries&nbsp;=&nbsp;total\_products&nbsp;\*&nbsp;total\_providers
-</code>.
-The more products we add to the portfolio, or increase the support to other
-tools, doing this work by hand will not scale.
+Using Magic Modules to make changes to the generated tools might seem
```

Comment:
I'd get rid of this paragraph all-together. I think it might unintentionally scare off contributors. After all, thanks to new documentation, it shouldn't be that intimidating at all!